OnlyT/Services/TalkSchedule/TalkScheduleFileBased.cs

```diff
@@ -96,8 +97,8 @@ private static TimeSpan AttributeToDuration(XAttribute? attribute)
 
         private static TimeSpan StringToTimeSpan(string s)
```

Comment:
These changes are stylistic only and best removed so that the PR focuses on functional changes only.